carma_wm_ctrl/src/WMBroadcaster.cpp

```diff
@@ -1716,9 +1722,9 @@ cav_msgs::TrafficControlRequest WMBroadcaster::controlRequestFromRoute(const cav
 
   tcr_polygon_ = composeTCRStatus(localPoint, cB, local_projector); // TCR polygon can be visualized in UI
 
-  cB.oldest =ros::Time::now(); // TODO this needs to be set to 0 or an older value as otherwise this will filter out all controls
+  cB.oldest =rclcpp::Time(0.0,0.0,RCL_SYSTEM_TIME); // TODO this needs to be set to 0 or an older value as otherwise this will filter out all controls
```

Comment:
is RCL_SYSTEM_TIME the right default? What if we are running in simulation?

Reply:
getting clocktype from scheduler now